app/controllers/health/appointments_controller.rb

```diff
@@ -17,9 +17,14 @@ def index
       set_hpc_patient
       # set_patient if @patient.blank?
       a_t = Health::Appointment.arel_table
-      @appointments = @patient.appointments.order(appointment_time: :desc)
+      @appointments = if @patent.present?
```

Comment:
Maybe you meant @patient?
```suggestion
      @appointments = if @patient.present?
```